### PR TITLE
Disallow pasting media into editor3

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDFID-358

### CODE REVIEW

File: scripts/core/editor3/components/handlePastedText.js [hunk 1]
```diff
@@ -19,7 +31,13 @@ const NOT_HANDLED = 'not-handled';
  * @description Handles pasting into the editor, in cases where the content contains
  * atomic blocks that need special handling in editor3.
  */
-export function handlePastedText(editorKey, text, html) {
+export function handlePastedText(editorKey, text, _html) {
+    var html = _html;
+
+    if (typeof html === 'string') {
+        html = removeMediaFromHtml(html);
```
[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/core/editor3/components/handlePastedText.js]: No it doesn't. Was this ever a requirement?

[the remark this answers is not in the document]
Comment (ioanpocol) [reply scripts/core/editor3/components/handlePastedText.js]: I think yes, but better ask Karel about it.

[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/core/editor3/components/handlePastedText.js]: I've discussed it with Karel - we don't need to support copy-pasting images within the editor

[the remark this answers is not in the document]
Comment (ioanpocol) [reply scripts/core/editor3/components/handlePastedText.js]: ok

**Pull request merged**